app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class BooksController < ApiV1Controller
+      def index
+        render_paginated Book
+      end
+
+      def show
+        render json: Book.find(params[:id])
```

Comment:
We use to require parameters to be sure frontend or external service send well information. Also you can define a method to do the find, like that:
```ruby
private

def book
  Book.find(params.require(:id))
end
```

2 more points: can you hanlde when the require raise an error o the find?